install_config/upgrades.adoc

```diff
@@ -687,16 +742,18 @@ endif::[]
 ifdef::openshift-origin[]
 ====
 ----
-# oc create -n openshift -f /usr/share/openshift/examples/image-streams/image-streams-centos7.json
-# oc create -n openshift -f /usr/share/openshift/examples/db-templates
-# oc create -n openshift -f /usr/share/openshift/examples/quickstart-templates
-# oc replace -n openshift -f /usr/share/openshift/examples/image-streams/image-streams-centos7.json
-# oc replace -n openshift -f /usr/share/openshift/examples/db-templates
-# oc replace -n openshift -f /usr/share/openshift/examples/quickstart-templates
+# oc create -n openshift -f roles/openshift_examples/files/examples/v1.1/image-streams/image-streams-centos7.json
+# oc create -n openshift -f roles/openshift_examples/files/examples/v1.1/db-templates
+# oc create -n openshift -f roles/openshift_examples/files/examples/v1.1/quickstart-templates
+# oc replace -n openshift -f roles/openshift_examples/files/examples/v1.1/image-streams/image-streams-centos7.json
+# oc replace -n openshift -f roles/openshift_examples/files/examples/v1.1/db-templates
+# oc replace -n openshift -f roles/openshift_examples/files/examples/v1.1/quickstart-templates
```

Comment:
Not currently, lets discuss tomorrow.

On Mon, Nov 23, 2015 at 5:23 PM, Alex Dellapenta <notifications@github.com>
wrote:

> In install_config/upgrades.adoc
> <https://github.com/openshift/openshift-docs/pull/1251#discussion_r45670004>
> :
>
> > @@ -687,16 +742,18 @@ endif::[]
> >  ifdef::openshift-origin[]
> >  ====
> >  ----
> > -# oc create -n openshift -f /usr/share/openshift/examples/image-streams/image-streams-centos7.json
> > -# oc create -n openshift -f /usr/share/openshift/examples/db-templates
> > -# oc create -n openshift -f /usr/share/openshift/examples/quickstart-templates
> > -# oc replace -n openshift -f /usr/share/openshift/examples/image-streams/image-streams-centos7.json
> > -# oc replace -n openshift -f /usr/share/openshift/examples/db-templates
> > -# oc replace -n openshift -f /usr/share/openshift/examples/quickstart-templates
> > +# oc create -n openshift -f roles/openshift_examples/files/examples/v1.1/image-streams/image-streams-centos7.json
> > +# oc create -n openshift -f roles/openshift_examples/files/examples/v1.1/db-templates
> > +# oc create -n openshift -f roles/openshift_examples/files/examples/v1.1/quickstart-templates
> > +# oc replace -n openshift -f roles/openshift_examples/files/examples/v1.1/image-streams/image-streams-centos7.json
> > +# oc replace -n openshift -f roles/openshift_examples/files/examples/v1.1/db-templates
> > +# oc replace -n openshift -f roles/openshift_examples/files/examples/v1.1/quickstart-templates
>
> Do we package a-o-utils & friends for Origin?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/openshift-docs/pull/1251/files#r45670004>.
>